src/decode.h

```diff
@@ -778,6 +779,126 @@ void CaptureStatsSetup(ThreadVars *tv, CaptureStats *s);
         }                                           \
     } while(0)
 
+/**
+ *  \brief Initialize a packet structure for use.
+ */
+#define PACKET_INITIALIZE(p)                                                                       \
```

Comment:
why are these reintroduced? Replaced by `PacketInit` and friends in `src/packet.[ch]`

Reply:
My mistake; will correct.